src/Albany_RegressionTests.cpp

```diff
@@ -172,17 +166,15 @@ RegressionTests::getTestParameters(int response_index) const
 {
   Teuchos::ParameterList* result;
 
-  if (response_index == 0 && appParams->isSublist("Regression Results")) {
-    result = &(appParams->sublist("Regression Results"));
-  } else {
-    result = &(appParams->sublist(
-        strint("Regression Results", response_index)));
-  }
+
+  result = &(appParams->sublist(
+      strint("Regression For Response", response_index)));
+
 
   TEUCHOS_TEST_FOR_EXCEPTION(
-      result->isType<std::string>("Test Values"),
+      result->isType<std::string>("Test Value"),
```

Comment:
Probably this is not needed, since we would get an error anyway if the Type is wrong.